### PR TITLE
[IOS] fix tests for Image upgrade

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

fix tests for an Image upgrade
IOS v15.6.2(T) - IOSXE C8000v v17.06.02

##### ISSUE TYPE

- Test fix Pull Request

##### COMPONENT NAME

integration tests

##### ADDITIONAL INFORMATION

- [x] ios_banner - Remove net_banenr tests as deprecated in netcommon
- [x] ios_interface - Remove as deprecated
- [x] ios_l2_interface - Remove as deprecated
- [x] ios_l3_interface - Remove as deprecated
- [x] ios_vlan - Remove as deprecated
- [x] ios_static_route - Remove as deprecated
- [x] ios_logging_global - Fix test, remove deprecated attribute as per appliance
- [x] ios_bgp_global - Fix test, update valid playbook attribute
- [x] ios_lldp - Remove provider check for connection timeout
- [x] ios_logging - Remove provider check for connection timeout
- [x] ios_ntp - Remove provider check for connection timeout
- [x] ios_ping - Remove provider check for connection timeout
- [x] ios_smoke - Fix test, update valid playbook attribute
- [x] ios_system - Fix test, update valid playbook attribute

```paste below

```

### CODE REVIEW

File: tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml [hunk 1]
```diff
@@ -52,8 +52,6 @@
             tag: hostname
           persistent:
             batch: 4444
-          policy_firewall:
```
Comment (NilashishC) [inline tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml:55]: Are these attributes deleted?

File: tests/integration/targets/ios_logging_global/vars/main.yaml [hunk 2]
```diff
@@ -49,8 +48,6 @@ merged:
       - log
     monitor:
       severity: warnings
-    policy_firewall:
```
Comment (NilashishC) [inline tests/integration/targets/ios_logging_global/vars/main.yaml:52]: Same as above.

File: tests/integration/targets/ios_banner/tests/cli/net_banner.yml [hunk 3]
```diff
@@ -1,29 +0,0 @@
----
```
Comment (NilashishC) [inline tests/integration/targets/ios_banner/tests/cli/net_banner.yml:1]: Why are we removing net_banner tests?

Comment (KB-perByte) [reply tests/integration/targets/ios_banner/tests/cli/net_banner.yml:1]: https://github.com/ansible-collections/ansible.netcommon/blob/dde94450cc9cc6d07a52b98c1eec8a1fcfc5240c/plugins/modules/net_banner.py#L13
it's deprecated and tobe removed from netcommon in the upcoming release

File: tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml (hunk 1 above)
Comment (KB-perByte) [reply tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml:55]: so with the image version upgrade, specific to the l3 image, this is an invalid attribute.
We might want to deprecate the attribute if it's not valid for l2 too, we would need to wait before we have an image with us.

Comment (NilashishC) [reply tests/integration/targets/ios_logging_global/tests/cli/rendered.yaml:55]: Okay, got it.

**Review by NilashishC — APPROVED** [review]

**Pull request merged**